library/ecp_invasive.h

```diff
@@ -40,8 +40,28 @@ typedef enum {
     MBEDTLS_ECP_MOD_SCALAR
 } mbedtls_ecp_modulus_type;
 
+/* Provide a commented-out definition so that `check_names.py` knows that
+ * it's not a typo.
+ * MBEDTLS_ECP_WITH_MPI_UINT must not be defined within config files, but
+ * only on the command line.
```

Comment:
```suggestion
/* Provide a commented-out definition so that `check_names.py` knows that
 * it's not a typo.
```
This part of the comment can be removed - it is safe (and allowed) to define it within config files, the brief was that it should not be in mbedtls_config.h YET, as this is not a finished feature, by any means.